student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
+    super()
```

Comment:
- [ ] Kindly add `age, name, parent_permission` as parameters for super ,so you can initialize student like this  `std=Student.new(age = 10, classroom = "math",name = "student1",parent_permission = true)`.
Your code will be look like this:
```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```